office365_connector.py

```diff
@@ -1132,6 +1138,17 @@ def _process_email_data(self, config, action_result, endpoint, email):
 
         return phantom.APP_SUCCESS
 
+    def _remove_tokens(self, action_result):
+        if len(list(filter(lambda x: x in action_result.get_message(), MSGOFFICE365_ASSET_PARAM_CHECK_LIST_ERRORS))) > 0:
```

Comment:
nit: should we add a short comment on the long condition?

I believe we are checking if the message includes any of the known error codes.

Reply:
updated